promisasu/Portal
# Merge for web app changes

**Taiga User Story**
Tg-597
**Taiga Task(s)**
Tg-654
**What did you change?**
Added a route for the web-application to be hosted from the node server using the path of the web-app  mentioned in config.json
**How can we test?**
1. run npm init
2. Check that config.json file is created.
3. Edit the config.json file and add the below after "database" object
``` json
"application": {
     "path": "<path pointing to the web-app part on the server"
 }
```
4. run npm start
5. Check that the node server is hosted successfully.
6. Check for the route "/promis" and see that the web-application is hosted.
